_posts/2016-01-30-iOS-Junior-Interviews.markdown

```diff
@@ -0,0 +1,192 @@
+---
+layout: post
+title: "Interviewing, applying and getting your first job in iOS"
+date: 2016-01-30 11:09
+author: orta
+categories: [mobile, ios, juniors, interviewing]
+---
+
+When I was at the beginning of my career, my first developer job application was to a design agency who were doing Mac development too. It was pretty nerve racking to apply for my first programming job, and I came into the interview with no idea what to expect.
+
+Now that I'm both further on in my career, and involved with so many juniors in NYC, I'm meeting with a lot of people who are in the same position and I get a lot of questions asking what they can do to prepare. This post _attempts_ at being a comprehensive collection of recommendations. It is subjective, of course, and strongly based towards my experiences.
+
+Before I jump through to the article, there's one thing that should be above the fold. _Chill out_. You might not get it right on the first try, I've applied for jobs and received a "no thanks." Yet eventually I got here.
```

Comment:
I like this, and I like that it's above the fold :heart_eyes_cat: I would add something like "So take a deep breath, it's going to be okay." to the end of the paragraph, but that's just style.